File: identicon/src/main/java/com/lelloman/identicon/ClassicIdenticonView.java
package com.lelloman.identicon;

import android.content.Context;
import android.util.AttributeSet;

import com.lelloman.identicon.classic.ClassicIdenticonDrawable;

public class ClassicIdenticonView extends IdenticonView {

	public ClassicIdenticonView(Context context) {
		super(context);
	}

	public ClassicIdenticonView(Context context, AttributeSet attrs) {
		super(context, attrs);
	}

	@Override
	protected IdenticonDrawable makeIdenticonDrawable(int width, int height, int hash) {
		return new ClassicIdenticonDrawable(width, height, hash);
	}
}

File: identicon/src/main/java/com/lelloman/identicon/view/ClassicIdenticonView.kt
package com.lelloman.identicon.view

import android.content.Context
import android.util.AttributeSet

import com.lelloman.identicon.drawable.ClassicIdenticonDrawable

class ClassicIdenticonView @JvmOverloads constructor(
    context: Context,
    attrs: AttributeSet? = null
) : IdenticonView(context, attrs) {
    override fun makeIdenticonDrawable(width: Int, height: Int, hash: Int) =
        ClassicIdenticonDrawable(width, height, hash)
}
